### PR TITLE
Try to also look for kubeadm repo from GOPATH

### PR DESCRIPTION
**Pull request by krzyzacy**
so it can support both path...
/assign @neolit123

### CODE REVIEW

**Comment by krzyzacy** [comment]
hope now it will be able to perform a bazel build... will see where it stops after this :-)

**Comment by neolit123** [comment]
this is still red:
https://k8s-testgrid.appspot.com/presubmits-kubernetes-nonblocking#pull-kubernetes-e2e-kubeadm-gce

```
2019/02/02 17:06:20 main.go:315: Something went wrong: error creating deployer: open /workspace/k8s.io/kubernetes-anywhere/.config: no such file or directory
```

could it be that the k-a repo is not cloned under `workspace`?
i would send a PR if i knew what how to fix it. :thinking:

**Comment by krzyzacy** [comment]
oh I haven't merge the bump yet

**Comment by neolit123** [comment]
ok, thanks.
i will watch testgrid and report later.

File: scenarios/kubernetes_e2e.py [hunk 1]
```diff
@@ -563,9 +563,13 @@ def main(args):
 
     if args.kubeadm:
         version = kubeadm_version(args.kubeadm, shared_build_gcs_path)
+        # try to look for k-a repo
+        kubeadm_path = os.path.join(workspace, 'k8s.io', 'kubernetes-anywhere')
+        go_path = os.environ.get('GOPATH', '')
+        if go_path and os.path.exists(os.path.join(go_path, 'k8s.io', 'kubernetes-anywhere')):
```
Comment (krzyzacy) [inline scenarios/kubernetes_e2e.py]: doh - missing a 'src' here after GOPATH...

**Pull request merged**